tests/unit/test_models.py

```diff
@@ -846,6 +848,139 @@ def test_contains_container_types(container_column, engine):
     assert renderer.rendered == expected
 
 
+def test_column_defaults():
+
+    # Set should send the new value
+    called = False
+    num_called = 0
+    received = {}
+
+    @object_modified.connect
+    def verify_expected(_, *, obj, column, value):
+        nonlocal num_called, called, received
+        called = True
+        num_called += 1
+        received['obj'] = obj
+        received['column'] = column
+        received['value'] = value
+
+    class ColumnDefaultModel(BaseModel):
+        id = Column(Integer, hash_key=True, default=12)
+
+    default_obj = ColumnDefaultModel()
+
+    assert called is True
+    assert num_called == 1
+    assert received["obj"] is default_obj
+    assert received["column"] is ColumnDefaultModel.id
+    assert received["value"] == 12
+
+
+def test_column_default_func():
+
+    # Set should send the new value
+    called = False
+    num_called = 0
+    received = {}
+
+    @object_modified.connect
```

Comment:
This is a good test, but there should also be one for the number of times the default func is called (I mention this in the `Column.default` property above)

Reply:
👍